configs/dc2.yaml

```diff
@@ -0,0 +1,51 @@
+facility_name: Rubin-LSST
+obs_collection: LSST.DC2
+collections: ["2.2i/runs/DP0.1"]
+use_butler_uri: False
+dataset_types:
+  raw:
+    dataproduct_type: image
+    dataproduct_subtype: lsst.raw
+    calib_level: 1
+    obs_id_fmt: "{records[exposure].obs_id}"
+    o_ucd: phot.count
+    access_format: image/fits
+    datalink_url_fmt: "http://datalink.org/{obs_id}"
+  calexp:
+    dataproduct_type: image
+    dataproduct_subtype: lsst.calexp
+    calib_level: 2
+    obs_id_fmt: "{records[visit].name}"
+    o_ucd: phot.count
+    access_format: image/fits
+    datalink_url_fmt: "http://datalink.org/{obs_id}"
+  deepCoadd_calexp:
+    dataproduct_type: image
+    dataproduct_subtype: lsst.coadd
```

Comment:
Is the subtype meant to be the LSST dataset type or some modified variant? ie do we want `lsst.coadd` here or `lsst.deepCoadd_calexp`? cc/ @gpdf

Reply:
I'd like the actual Butler DatasetType used.  It will help users make connections between what they are doing in the VO services and the Portal with what they are doing in notebooks.

We can have a discussion closer to DP1 about whether to fine-tune the actual pipeline DatasetType names for clarity.